src/main/grpc/com/scalar/admin/rpc/AdminGrpc.java

```diff
@@ -5,14 +5,14 @@
 /**
```

Comment:
the files below are regenerated by the Protocol Buffers plugin.
I guess we don't need to review them.